editor/plugins/animation_player_editor_plugin.cpp

```diff
@@ -1262,9 +1262,7 @@ void AnimationPlayerEditor::_seek_value_changed(float p_value, bool p_set, bool
 
 	if (!p_timeline_only) {
 		if (player->is_valid() && !p_set) {
-			double delta = player->get_current_animation_position();
```

Comment:
This particular line of code is actually not returning a delta but rather the actual current position. It will make `player->seek(pos + delta, true, true);` totally wrong.